src/item.py

```diff
@@ -0,0 +1,13 @@
+class Item():
+  def __init__(self, name , description):
+    self.name = name
+    self.descript = description
+
+  def on_take(self, item):
```

Comment:
The on take method should be in the Player class in order to add that item to the players items list. There should also be a remove item method in the Room class to remove that item from a room and provide it to the user via the on take method.